apps/docs/spec/supabase_py_v2.yml

```
@@ -515,36 +696,362 @@ functions:
+          - name: options
+            isOptional: true
+            type: object
+            subContent:
+              - name: captcha_token
+                isOptional: true
+                type: string
+          - name: nonce
+            isOptional: true
+            type: string
+            description: |
+              If the ID token contains a `nonce` claim, then the hash of this value is compared to the value in the ID token.
+          - name: access_token
+            isOptional: true
+            type: string
+            description: |
+              If the ID token contains an `at_hash` claim, then the hash of this value is compared to the value in the ID token.
+    notes: |
+      Allows signing in with an OIDC ID token. The authentication provider used should be enabled and configured.
+    examples:
+      - id: sign-in-with-id-token
+        name: 'Sign In using ID Token'
+        code: |
+          ```python
+          response = supabase.auth.sign_in_with_id_token(
+            {"provider": "google", "token": "your-id-token"}
+          )
+          ```
+        response: |
+          ```json
+          {
+            "session": {
+              "access_token": "<ACCESS_TOKEN>",
+              "expires_at": 1721236789,
+              "expires_in": 60,
+              "provider_refresh_token": null,
+              "provider_token": null,
+              "refresh_token": "<REFRESH_TOKEN>",
+              "token_type": "bearer",
+              "user": {
+                "action_link": null,
+                "app_metadata": {
+                  ...
+                },
+                "aud": "authenticated",
+                "confirmation_sent_at": null,
+                "confirmed_at": "Wed, 17 Jul 2024 16:34:24 GMT",
+                "created_at": "Wed, 17 Jul 2024 16:34:24 GMT",
+                "email": "email@example.com",
+                "email_change_sent_at": null,
+                "email_confirmed_at": "Wed, 17 Jul 2024 16:34:24 GMT",
+                "factors": null,
+                "id": "11111111-1111-1111-1111-111111111111",
+                "identities": [
+                  {
+                    "identity_id": "22222222-2222-2222-2222-222222222222",
+                    "provider": "google",
+                    "user_id": "11111111-1111-1111-1111-111111111111"
+                  }
+                ],
+                "invited_at": null,
+                "last_sign_in_at": "Wed, 17 Jul 2024 17:18:49 GMT",
+                "new_email": null,
+                "phone": "",
+                "phone_confirmed_at": null,
+                "recovery_sent_at": null,
+                "role": "authenticated",
+                "updated_at": "Wed, 17 Jul 2024 17:18:49 GMT",
+                "user_metadata": {
+                  ...
+                }
```

/nit  not really important - can fix later
```suggestion
                }
     "is_anonymous": "false",
```